app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1078,7 +1078,7 @@ public void onFailure(Call<SusiResponse> call, Throwable t) {
 
     private void updateDatabase(final long prevId, final String message, final boolean isDate, final String date, final String timeStamp , final boolean mine,
```

Comment:
@the-dagger nice idea. Can we configure circle ci script to build javadocs after every PR and push it to `gh-pages` branch?